WooCommerce/Classes/ViewRelated/Products/AI/Celebration/ProductDescriptionGenerationCelebrationView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+
+/// Hosting controller for `ProductDescriptionGenerationCelebrationView`.
+///
+final class ProductDescriptionGenerationCelebrationHostingController: UIHostingController<ProductDescriptionGenerationCelebrationView> {
+    init(viewModel: ProductDescriptionGenerationCelebrationViewModel) {
+        super.init(rootView: ProductDescriptionGenerationCelebrationView(viewModel: viewModel))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+/// Celebration view presented when AI generation is used for the first time
+struct ProductDescriptionGenerationCelebrationView: View {
+    private var viewModel: ProductDescriptionGenerationCelebrationViewModel
+
+    init(viewModel: ProductDescriptionGenerationCelebrationViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        VStack(spacing: Constants.verticalSpacing) {
+            Image(uiImage: viewModel.celebrationImage)
+
+            Text(viewModel.greatStartLabel)
+                .headlineStyle()
+                .multilineTextAlignment(.center)
+
+            Text(viewModel.instructionsLabel)
+                .foregroundColor(Color(.text))
+                .subheadlineStyle()
+                .multilineTextAlignment(.center)
```

Comment:
Super nit: I notice on the design that the padding for the text is 24. We probably want to follow it to make the text stay more condensed in the center.

Reply:
Good catch. Done in https://github.com/woocommerce/woocommerce-ios/pull/9939/commits/3b8310f5a37585f199a78ee904a985f21ffc0739